packages/langium/package.json

```diff
@@ -18,14 +18,43 @@
   "license": "MIT",
   "files": [
     "lib",
-    "src",
-    "test.js",
-    "test.d.ts"
+    "src"
   ],
```

Comment:
Shouldn't you now add:
```
"type": "module",
```
[new type in package.json](https://www.typescriptlang.org/docs/handbook/esm-node.html#type-in-packagejson-and-new-extensions). I guess we/I should have done that in other projects as well, but we didn't. According to the doc it should be the case.